src/platform/site-wide/va-footer/helpers.jsx

```diff
@@ -20,19 +20,20 @@ export const FOOTER_EVENTS = {
   CRISIS_LINE: 'nav-footer-crisis',
 };
 
-const renderInnerTag = (link, captureEvent) => [
-  link.label ? (
-    <span className="va-footer-link-label">{link.label}</span>
-  ) : null,
-
-  link.href ? (
-    <a href={link.href} onClick={captureEvent} target={link.target}>
-      {link.title}
-    </a>
-  ) : (
-    <span className="va-footer-link-text">{link.title}</span>
-  ),
-];
+const renderInnerTag = (link, captureEvent) => (
+  <>
```

Comment:
Is this `<>` meant to be here?

Reply:
I think that's [short syntax](https://reactjs.org/docs/fragments.html#short-syntax) for react fragments

Reply:
Yes, that's essentially the fix, so that React does not see a list and complain about no list-key.
<> ... </> is shorthand for [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax).